BUILD.bazel

```diff
@@ -1 +1,15 @@
 # TODO(#1532): Rename file to 'BUILD' post-Gradle.
+'''
+Oppia Android binary
+'''
+android_binary(
+    name = "oppia",
+    custom_package = "org.oppia.app",
+    enable_data_binding = True,
+    manifest = "//app:src/main/AndroidManifest.xml",
```

Comment:
Suggest adding TODO to move this manifest to be top-level alongside the binary target post-phase 1.

Reply:
Done!